components/scheduler/src/main/java/org/cardanofoundation/ledgersync/scheduler/service/offchain/ProcessService.java

```diff
@@ -0,0 +1,8 @@
+package org.cardanofoundation.ledgersync.scheduler.service.offchain;
+
+import org.cardanofoundation.ledgersync.consumercommon.entity.OffChainDataCheckpoint;
+
+public interface ProcessService {
```

Comment:
let's rename this class for clarity, example: GovOffChainDataService or something better